### PR TITLE
SC 1.2.2 Thorough Review

### PR DESCRIPTION
**Pull request by nitedog**
Addressing issues from [thorough review](https://www.w3.org/2002/09/wbs/1/WCAGvideos_batch1thorough/results#xq1):
- Moved indication of hearing aid from scene 3 to scene 1 for better clarity
- Clarified the necessity of conveying important audio information in scene 3
- Scene 4: focused the impact on the videos rather than on the course overall
- Improved explanation for automatically-generated captions in scene 5
- Other clarifications and wording improvements throughout

### CODE REVIEW

File: content/SC-1.2.2.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ This page provides the script for Success Criterion 1.2.2 Captions (Prerecorded)
...
-| 5 | 0:34 - 0:40 | Also automatic captioning is often inaccurate. For example, it does not recognize some words and accents. | Martine clicks (staged) auto-caption button, and garbled captions are displayed. |
-| 6 | 0:40 - 0:46 | Thankfully the university is now adding captions to all the videos that Martine has in her course. | Reverse of scene 1: view from behind Martine sitting at her desk and watching a video moves around to see frontal view of Martine's face -- she is happy/smiling. |
+| 1 | 0:00 - 0:09 | Martine is hard of hearing. She can hear some sounds with her hearing aid but not enough to understand speech. | Frontal view of Martine's face moving around and away from her, to see her from behind sitting at a desk and working on a computer. She has a hearing aid that is visible from this angle. (similar approach to [Captions - Perspectives Video](https://www.w3.org/WAI/perspective-videos/captions/)). |
+| 2 | 0:09 - 0:19 | She is currently taking an online college course. Most course videos have captions, which Martine uses to know what is being said. | Zoom in on computer screen to find a video playing. Zoom further to the captions appearing in the video. |
```
Comment (jade-mc) [inline content/SC-1.2.2.md:58]: Not sure why this is being changed to 'know', it isn't a natural collocation.

File: content/SC-1.2.2.md [hunk 2]
```diff
@@ -54,9 +54,9 @@ This page provides the script for Success Criterion 1.2.2 Captions (Prerecorded)
...
+| 3 | 0:19 - 0:27 | The captions also convey important audio information, such as who is speaking, clapping, and laughter. This helps convey the context and overall tone in the audio. | Clapping and switch of speakers indicated in the captions, reflecting the video playing. |
+| 4 | 0:27 - 0:34 | Unfortunately some videos do not have captions, making them not usable for Martine. | Martine opens another video and tries clicking the disabled closed captions button. Martine looks disappointed. |
+| 5 | 0:34 - 0:40 | Also videos with automatically-generated captions are not usable for Martine. They are generally too inaccurate and sometimes wrong. | Martine turns on automatic captions, and we see a captioned sentence that obviously does not make sense. |
```
Comment (jade-mc) [inline content/SC-1.2.2.md]: 'sometimes wrong' is the same as 'generally too inaccurate' surely? Perhaps get rid of 'sometimes wrong', suggestion: _They are generally too inaccurate for her to rely on._

Comment (bakkenb) [reply content/SC-1.2.2.md]: +1 to Jade's comment here.

File: content/SC-1.2.2.md (hunk 1 above)
Comment (bakkenb) [reply content/SC-1.2.2.md:58]: I am okay with changing this to "know." To me this is more accurate. Captions don't help you "understand" what is being said, they simply help you "know" what is being said.

File: content/SC-1.2.2.md [hunk 3]
```diff
@@ -54,9 +54,9 @@ This page provides the script for Success Criterion 1.2.2 Captions (Prerecorded)
...
+| 6 | 0:40 - 0:46 | Thankfully the university is now adding proper captions to all the videos that Martine has in her course. | Reverse of scene 1: view from behind Martine sitting at her desk and watching a video moves around to see frontal view of Martine's face -- she is engaged with the video. |
```
Comment (bakkenb) [inline content/SC-1.2.2.md]: +1 to this change.

File: content/SC-1.2.2.md (hunk 1 above)
Comment (nitedog) [reply content/SC-1.2.2.md:58]: Keeping "know" for now, for the reasons that Brent explained.

**Pull request merged**